mirrord/config/src/feature/fs/advanced.rs

```diff
@@ -19,6 +19,9 @@ use crate::{
 /// 3. `"local"` - Read from the local file system.
 /// 5. `"disable"` - Disable file operations.
```

Comment:
I think we're missing number 4 here? Is it localwithoverrides?

Reply:
I think you're right. Also, I don't think there is a `disable` mode.
```suggestion
/// 4. `"localwithoverrides"` - perform fs operation locally, unless the path matches a pre-defined or user-specified exception.
```